### ISSUE TITLE
Create a "easystats: the essential functions" vignette

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/easystats#228]
we could create here in easystats a vignette about the star-features.

functions to showcase:

- `check_model()`
- `performance()` and `parameters()` (or if @strengejacke finally helps me make a master function that integrates the two, that'd be awesome ^^), where we need to emphasize that it encapsulates all the features from bayestestR and effectsize
- `estimate_relation()`
- `report()`
- `correlation()`
- ...

**Comment by DominiqueMakowski** [comment]
tagging @etiennebacher if he's interested as this vignette could be something that we could advertise on Twitter and all and could bring some attention to our newest member 😉

**Comment by etiennebacher** [comment]
Hi, unfortunately I don't use {easystats} enough to know its main features and functions (I just help fixing a few bugs here and there and adding some features to datawizard)

**Comment by strengejacke** [comment]
> * or if @strengejacke finally helps me make a master function that integrates the two, that'd be awesome ^^

Yeah, something like https://github.com/easystats/report/issues/65 would be great! :-)

**Comment by strengejacke** [comment]
Yeah, I think having vignettes in general would be great, but also important for https://github.com/easystats/easystats/issues/113#issuecomment-820392645.

**Comment by IndrajeetPatil** [comment]
I am not sure how useful this will be for the users but the Pareto principle, when applied to software development, dictates that 80% of user’s easystats runtime will be due to 20% of the functions, which is precisely this list.

I think such a list would provide us a good idea as to which functions we should be optimizing after the stable release to improve user experience the most.
